community/portworx/templates/csi-crd.yaml

```diff
@@ -0,0 +1,700 @@
+{{- if (.Values.csi) and (eq .Values.csi true)}}
```

Comment:
I think you need to put the CRDs in a separate folder so that they are applied before the other specs. Also, doing it this way, if users uninstall the helm chart it'll delete the CRDs which will cause all their snapshots (even non-PX ones) to get deleted

Reply:
If all these CRD's are for Auto-Pilot, can we bring them in once we bring auto-pilot to IBM cloud?